src/widgets/support/ui/support.tsx

```diff
@@ -19,11 +22,22 @@ export const Support = () => (
           events. Every donation, big or small, helps!
         </Paragraph>
         <Paragraph fontSize="large">Thank you for your support!</Paragraph>
-        <LinkCustom href={LINKS.DONATE} variant="primary" external>
+        <LinkCustom
+          className={cx('support-link')}
+          href={LINKS.DONATE}
+          variant="primary"
+          external
+          data-testid="link-custom"
```

Comment:
having the same data-testid can lead to confusion
![image](https://github.com/user-attachments/assets/8e264aba-9281-4edd-b057-f96c8ce96870)

Although this is not an issue since only `Support` component is rendered in the test, I'd still update those data-testid attributes

Reply:
It would be better if there was a data-testid on the `LinkCustom` component but it was not added. I renamed it, thanks.